OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseOutlineFragment.java

```diff
@@ -180,8 +184,9 @@ public void onRefresh() {
         });
         UiUtil.setSwipeRefreshLayoutColors(swipeContainer);
         restore(bundle);
+        calendarTitle = environment.getConfig().getPlatformName() + " - " + courseData.getCourse().getName();
+        accountName = environment.getLoginPrefs().getCurrentUserProfile().name;
         initListView(view);
-        initObserver();
```

Comment:
need to revert this change otherwise multiple `Observer` will add on the pull to refresh. and catch the response multiple times.

Reply:
Done